es7-persistence/src/main/java/com/netflix/conductor/es7/config/ElasticSearchProperties.java

```diff
@@ -89,6 +89,17 @@
     int PRUNING_BATCH_SIZE_DEFAULT_VALUE = 2000;
     int PRUNING_DAYS_TO_KEEP_DEFAULT_VALUE = 28; // 4 weeks
 
+    /** Enable AWS ES */
+    private boolean aws = false;
+
+    public boolean getAws() {
+        return aws;
+    }
+
+    public void setAws(boolean aws) {
```

Comment:
where is this being called based on configuration?

Reply:
This will read from https://bitbucket-eng-sjc1.cisco.com/bitbucket/projects/AN/repos/galaxy/browse/docker/config/dynomite-cluster.config.properties